src/Hero6.UserInterface.SierraVga/Entities/Padding.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LateStartStudio.Hero6.UserInterface.SierraVga.Entities
+{
+    public struct Padding
+    {
+        public float Left { get; set; }
```

Comment:
I can't really say I share your preference on this, couple of things

1. StyleCop is suggesting it, obviously.
2. Putting getters and setters after the constructor, but before the functions, isn't a C# thing, it's the normalized style rule for all OOP languages. By following this pattern we have a style that's easy to adopt for any other regardless of where they're coming from.
3. It sounds like you're suggesting we could put properties in two separate places depending on their implementation, which I don't approve of at all, this is almost a text book example of organizing code in such fashion which only makes sense to the individual that thought of it.
4. The fact that properties can, and some probably will implement code, is to me a reason enough that properties should go after the constructor.

I feel if I sat down and thought about this for a while I would have lots more to say about this, however this is quickly becoming a wall of text. Conclusively I can only say that I don't share your sentiment that properties before the constructor would be more readable, although I can certainly see the logic behind it.